utils/billgeneration.js

```diff
@@ -1,20 +1,68 @@
 /* eslint-disable import/imports-first */
 import moment from 'moment';
 import dotenv from 'dotenv-safe';
+import fs from 'fs';
 
+import React from 'react';
+import ReactPDF from '@react-pdf/renderer';
 import {
   getSolarProjectById,
   getOwnerById,
   getRateScheduleById,
   createSubscriberBill,
   updateSubscriberBill,
-  getSubscriberBillsByStatementNumber
+  getSubscriberBillsByStatementNumber,
+  getSubscriberBillById
 } from '../airtable/request';
 import getEnphaseData from './enphase';
 import getLatestBill from './utilityApi';
+import BillingTemplate from './pdf/BillingTemplate';
+
+// const ROOT_URL = 'https://1f7c24d2.ngrok.io';
```

Comment:
what was this? I'd remove it if it's nthing too significant

Reply:
it's a local testing url, but yup sure good point